test/e2e/preflightCliTest_test.go

```diff
@@ -0,0 +1,128 @@
+package e2e
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestPreflightHelp(t *testing.T) {
+	got := preflightHelp()
+	want := "A utility that allows you to pre-test your bundles, operators, and container before submitting for Red Hat Certification.\n\nUsage:\n  preflight [command]\n\nAvailable Commands:\n  certify        Submits check results to Red Hat\n  check          Run checks for an operator or container\n  completion     generate the autocompletion script for the specified shell\n  help           Help about any command\n  runtime-assets Returns information about assets used at runtime.\n  support        Submits a support request\n\nFlags:\n  -h, --help      help for preflight\n  -v, --version   version for preflight\n\nUse \"preflight [command] --help\" for more information about a command.\n"
+
+	if got != want {
+		t.Errorf("got %q \n\n\n\nwant %q", got, want)
+	}
+}
+
+func TestPreflightCheck(t *testing.T) {
+	got := preflightCheck()
+	want := "This command will allow you to execute the Red Hat Certification tests for an operator or a container.\n\nUsage:\n  preflight check [command]\n\nAvailable Commands:\n  container   Run checks for a container\n  operator    Run checks for an Operator\n\nFlags:\n  -h, --help   help for check\n\nUse \"preflight check [command] --help\" for more information about a command.\n"
+
+	if got != want {
+		t.Errorf("got %q \n\n\n\nwant %q", got, want)
+	}
+}
+
+func TestPreflightCheckContainer(t *testing.T) {
+	got := preflightCheckContainer()
+	want := "Error: not enough positional arguments: A container image positional argument is required\n\nThe checks that will be executed are the following:\n-"
+
+	container_checks := []string{"LayerCountAcceptable", "HasNoProhibitedPackagesMounted", "HasRequiredLabel", "RunAsNonRoot", "BasedOnUbi", "HasLicense", "HasUniqueTag"}
```

Comment:
I believe you can get the list of checks by running:
```./preflight check container --list-checks```
It is better not to hard code these values.